streampipes-extensions-management/src/main/java/org/apache/streampipes/extensions/management/connect/adapter/preprocessing/elements/SendToBrokerAdapterSink.java

```diff
@@ -86,9 +87,12 @@ protected void sendToBroker(byte[] event) throws RuntimeException {
     producer.publish(event);
   }
 
-  protected abstract EventProducer makeProducer(T protocol);
-
-  public abstract void modifyProtocolForDebugging(T protocol);
+  public void modifyProtocolForDebugging(TransportProtocol protocol) {
+    protocol.setBrokerHostname("localhost");
```

Comment:
Does this also work in a Kubernetes deployment?

Reply:
see above, we don't support Kubernetes as development environment currently